repositories.yaml

```diff
@@ -2091,3 +2091,15 @@ applications:
     moz_pipeline_metadata_defaults:
       expiration_policy:
         delete_after_days: 400
+
+  - app_name: mozilla_backstage
+    canonical_app_name: Mozilla Service Catalog
+    app_description: Mozilla Service Catalog (Backstage)
+    url: https://github.com/mozilla-services/moz-backstage-app
+    notification_emails:
+      - wstuckey@mozilla.com
+    metrics_files:
+      - plugins/glean/metrics.yaml
+    channels:
+      - v1_name: moz-backstage
+        app_id: moz_backstage
```

Comment:
We don't yet require this value in CI/make it required in the schema but you need to set retention for this app e.g.:
```yaml
    moz_pipeline_metadata_defaults:
      expiration_policy:
        delete_after_days: 400
```